### PR TITLE
[W12H02] Fixed issues

### PR DESCRIPTION
**Pull request by MagnusSaurbier**
Changes:
-MultiStackSyncTest: use TimeoutChecker to automatically stop the test after Timeout is reached
-RingbufferMultithreadTest: Fixed issue, when test ran forever instead of terminating after timeout threshold
-MultistackMultithreadTest: Remove useless println in Singlethreadtest

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```
Comment (nilsreichardt) [inline .vscode/settings.json:5]: I think this shouldn't be in the PR, right?

File: w12h02/test/pgdp/ds/MultyStackSyncTests.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class MultiStackSyncTests {
```
Comment (nilsreichardt) [inline w12h02/test/pgdp/ds/MultyStackSyncTests.java:15]: Java Syntax error:
> The public type MultiStackSyncTests must be defined in its own file

File name needs to be changed

**Review by nilsreichardt — APPROVED** [review]
LGTM beside the two comments

**Comment by mbmueller** [comment]
can be closed

**Comment by nilsreichardt** [comment]
Closing this PR because the deadline is over

**Pull request closed without merging**